source/enemies/ck-meteorknight.js

```diff
@@ -47,17 +47,21 @@ module.exports = new EnemyTemplate("Meteor Knight",
 	selector: selectRandomFoe,
 	next: "random"
 }).addAction({
-	name: "Call Asteroid",
-	essence: "Unaligned",
-	description: "Summon an Asteroid, gain protection on Critical",
+	name: "Meteor Mayhem",
+	essence: "Fire",
```

Comment:
I've been setting the essence of actions that don't deal direct damage as Unaligned. I'm still pretty on the fence about that though (and to further the lack of tie-breaker, we don't have any mechanics that reference enemy action type yet either).

Mostly a note to decide on a convention and do a pass to ensure consistency.